operator/config/default/webhookcainjection_patch.yaml

```diff
@@ -21,6 +21,7 @@ webhooks:
             - "kube-system"
             - "kube-public"
             - "kube-node-lease"
+            - "cert-manager"
```

Comment:
is also this required?

Reply:
if you try to install the operator before the certificate manager those pods will be stuck and you will have to remove manually the mutatingwebhookconfig